docstore/example_test.go

```diff
@@ -356,7 +356,7 @@ func ExampleQuery_Update_full() {
 }
 
 func Example_optimisticLocking() {
-	// This example is used in https://gocloud.dev/howto/docstore.
+	// This example is used in https://gocloud.dev/howto/docstore#docrev.
```

Comment:
This will need updating if you update the alias in the main page as per review comment.

Reply:
This is the correct one.

Reply:
Done.